pkgs/development/python-modules/changefinder/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, wheel
+, numpy
+, scipy
+, statsmodels
+, nose
+}:
+
+buildPythonPackage rec {
+  pname = "changefinder";
+  version = "unstable-2024-03-24";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "shunsukeaihara";
+    repo = "changefinder";
+    rev = "58c8c32f127b9e46f9823f36221f194bdb6f3f8b";
+    hash = "sha256-1If0gIsMU8673fKSSHVMvDgR1UnYgM/4HiyvZJ9T6VM=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+    wheel
```

Comment:
Not sure where this gets recommended but to my knowledge we don't need to add wheel here.


```suggestion
```